src/lib.rs

```diff
@@ -42,6 +43,7 @@ pub enum ReleaseType {
     SafenodeManager,
     SafenodeManagerDaemon,
     SafenodeRpcClient,
+    SnAuditor,
```

Comment:
Is `SnAuditor` the actual binary name? Should we maybe just use `auditor`? (I would say `safeauditor`, but obviously we'll need to ditch all the `safe` prefixes soon.)

Reply:
Yes that is right. I just made sure to stay consistent with the bin name for now.